src/lib/Wst/Offchain/Endpoints/Deployment.hs

```diff
@@ -216,7 +216,17 @@ seizeCredentialAssetsTx :: forall era env m.
 seizeCredentialAssetsTx sanctionedCred = do
   opPkh <- asks (fst . Env.bteOperator . Env.operatorEnv)
   directory <- Query.registryNodes @era
-  seizeTxo <- head <$> Query.userProgrammableOutputs sanctionedCred
+  let getTxOutValue (C.TxOut _a v _d _r) = v
```

Comment:
Just selects the UTxO with the largest non ada asset quantities ignoring the asset names.
To seize all utxos the endpoints must be repeatedly called until `NoTokensToSeize` is returned.